src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```
@@ -2,13 +2,27 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exeption.InvalidInputException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
+
     @Override
     public User register(User user) {
```

The method should first check if the `user` object itself is null. Also, fields like `login`, `password`, and `age` can be null. Without checks, `user.getLogin()` would throw a `NullPointerException`, which is forbidden by checklist item #13. You should add checks for these null cases and throw your custom `InvalidInputException`.